client/src/mocks/stub-new-work/applications.ts

```diff
@@ -26,14 +26,29 @@ export const mockApplicationArray: Application[] = [
     ],
     binary: "app1-bin.zip",
     migrationWave: { id: 501, name: "Wave 1" },
-    assessments: [],
+    assessments: [{ id: 43, name: "test" }],
   },
 ];
 
 export const handlers = [
-  // rest.get(AppRest.APPLICATIONS, (req, res, ctx) => {
-  //   return res(ctx.json(mockApplicationArray));
-  // }),
+  // Commented out to avoid conflict with the real API
```

Comment:
Did you mean to commit these uncommented or comment them back out?

Reply:
Fixed. Thanks.